### PR TITLE
feat (spoke_mo): use incremental logic to update base tables

### PR DESCRIPTION
**Pull request by emily-flambe**
`dbt run` and `dbt test` work locally for me using the prod data with the failing tests (after deleting the dupe from `_airbyte_raw_message` and `message_base`, see Slack).

### CODE REVIEW

File: dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql
Comment (jitoquinto) [inline dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql:1]: looks like this one needs the partitions removed at the top!

**Review by jitoquinto — COMMENTED** [review]
looks like the last model needs to have the partitions removed but all else lgtm!

File: dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql
Comment (emily-flambe) [reply dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql:1]: ope actually zip_code is the one stream in this sync that IS full refresh overwrite, so it gets to go be alone in a 1_cta_full_refresh folder! Gonna make that change presently

Comment (emily-flambe) [reply dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql:1]: ok what do we think about this commit? https://github.com/community-tech-alliance/dbt-cta/pull/213/commits/3ce43b421acef1ef3afb40f512cbd1d962912a45

just ran these commands locally and they succeeded: 

```
pipenv run dbt run --target cta --select 1_cta_full_refresh
pipenv run dbt test --target cta --select 1_cta_full_refresh
```

**Review by kanelouise — APPROVED** [review]
LGTM, this special snowflake i think was the reason we stuffed them all in the same folder at first.

File: dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql
Comment (jitoquinto) [reply dbt-cta/spoke_mo/models/1_cta_incremental/zip_code_base.sql:1]: nice! :shipit:

**Review by jitoquinto — APPROVED** [review]

**Comment by emily-flambe** [comment]
> LGTM, this special snowflake i think was the reason we stuffed them all in the same folder at first.

yeah lol it's definitely unusual to have a sync with ONE stream that's full refresh while the others aren't. Computers 🫠

**Pull request merged**